provider/semgrep/README.md

```diff
@@ -0,0 +1,24 @@
+# Hello World context provider for OpenCtx
+
+This is a sample context provider for [OpenCtx](https://openctx.org) that fetches the contents of a Semgrep finding from a given URL.
+
+## Usage
+
+Add the following to your settings in any OpenCtx client:
+
+```json
+"openctx.providers": {
+    // ...other providers...
+    "https://openctx.org/npm/@openctx/provider-semgrep": {
```

Comment:
@sqs can you help me publish this package under https://openctx.org/npm/@openctx ?